src/command_modules/azure-cli-maps/azure/cli/command_modules/maps/_params.py

```diff
@@ -38,6 +38,11 @@ def load_arguments(self, _):
         c.argument('tags',
                    arg_type=tags_type)
 
+    with self.argument_context('maps account create') as c:
+        c.argument('force',
+                   action='store_true',
+                   help='Agree to the Terms of Service, and do not prompt for confirmation.')
```

Comment:
In the long summary for this parameter, you should include a link to said TOS (this requires an entry in help.py---cannot be done from this file).